[0 : output] Output to be read-from/written-into folder '../chains', with prefix 'nulcdm-pactl'
[1 : exception handler] ---------------------------------------

Traceback (most recent call last):
  File "/users/smp24dhl/.conda/envs/cosmos/bin/cobaya-run", line 8, in <module>
    sys.exit(run_script())
  File "/users/smp24dhl/.conda/envs/cosmos/lib/python3.9/site-packages/cobaya/run.py", line 198, in run_script
    run(info, **arguments.__dict__)
  File "/users/smp24dhl/.conda/envs/cosmos/lib/python3.9/site-packages/cobaya/run.py", line 123, in run
    with Model(updated_info["params"], updated_info["likelihood"],
KeyError: 'likelihood'
-------------------------------------------------------------

[2 : exception handler] ---------------------------------------

Traceback (most recent call last):
  File "/users/smp24dhl/.conda/envs/cosmos/bin/cobaya-run", line 8, in <module>
    sys.exit(run_script())
  File "/users/smp24dhl/.conda/envs/cosmos/lib/python3.9/site-packages/cobaya/run.py", line 198, in run_script
    run(info, **arguments.__dict__)
  File "/users/smp24dhl/.conda/envs/cosmos/lib/python3.9/site-packages/cobaya/run.py", line 123, in run
    with Model(updated_info["params"], updated_info["likelihood"],
KeyError: 'likelihood'
-------------------------------------------------------------

[3 : exception handler] ---------------------------------------

Traceback (most recent call last):
  File "/users/smp24dhl/.conda/envs/cosmos/bin/cobaya-run", line 8, in <module>
    sys.exit(run_script())
  File "/users/smp24dhl/.conda/envs/cosmos/lib/python3.9/site-packages/cobaya/run.py", line 198, in run_script
    run(info, **arguments.__dict__)
  File "/users/smp24dhl/.conda/envs/cosmos/lib/python3.9/site-packages/cobaya/run.py", line 123, in run
    with Model(updated_info["params"], updated_info["likelihood"],
KeyError: 'likelihood'
-------------------------------------------------------------

[0 : exception handler] ---------------------------------------

Traceback (most recent call last):
  File "/users/smp24dhl/.conda/envs/cosmos/bin/cobaya-run", line 8, in <module>
    sys.exit(run_script())
  File "/users/smp24dhl/.conda/envs/cosmos/lib/python3.9/site-packages/cobaya/run.py", line 198, in run_script
    run(info, **arguments.__dict__)
  File "/users/smp24dhl/.conda/envs/cosmos/lib/python3.9/site-packages/cobaya/run.py", line 123, in run
    with Model(updated_info["params"], updated_info["likelihood"],
KeyError: 'likelihood'
-------------------------------------------------------------

srun: error: node022: tasks 0-3: Exited with exit code 1
